stop.sh

```
@@ -0,0 +1,36 @@
+if systemctl --user is-active server-services-manager &>/dev/null; then
+    echo "Stopping systemd service..."
+    systemctl --user stop server-services-manager
+    echo "Server stopped."
+    exit 0
+fi
+
+if [ -f "$PIDFILE" ]; then
+    PID=$(cat "$PIDFILE")
+    echo "Stopping server (PID: $PID)..."
+    kill "$PID" 2>/dev/null
+    for i in $(seq 1 10); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)